app.js

```diff
@@ -0,0 +1,128 @@
+'use strict'
+
+
+var el = document.getElementById('sales-estimates');
+var firstAndPike = {
+  minCust: 23,
+  maxCust: 65,
+  avgSale: 6.3,
+  numCookies: [],
+  totalSales: 0,
+  hour: 5,
+  customerNumber: function(){
+    el.innerHTML += '<h3>1st and Pike</h3> \n <ul>Estimated Sales</ul>';
```

Comment:
You should avoid using innerHTML and instead use document.createElement() + el.textContent.   innerHTML allows you to inject javascript directly into html from outside, generally this should be avoided.